### PR TITLE
Add error handling for position details screen

### PR DESCRIPTION
**Pull request by mjoyce91**
Completes TM-552 by adding a 404 error message to the position details page if the position is not found.

### CODE REVIEW

**Comment by burgwyn** [comment]
screen shot 2019-03-01 at 2 10 39 pm

This works, but do want to consider a call to action in the error?

**Comment by mjoyce91** [comment]
@cydneycappello is there a better call to action we could put here?

**Comment by mjoyce91** [comment]
Based on discussion with @cydneycappello, I will add a link to the results page and add the "Back" button.

**Comment by mjoyce91** [comment]
See updates

File: src/sass/_details.scss [hunk 1]
```diff
@@ -103,6 +103,23 @@
     font-weight: bold;
   }
 
+  .position-error {
+    display: flex;
```
Comment (burgwyn) [inline src/sass/_details.scss]: 💪

**Pull request merged**